Source/TailwindTraders.Mobile/TailwindTraders.UITests/AppManager.cs

```diff
@@ -62,9 +62,9 @@ public static void StartApp()
             {
                 app = ConfigureApp
                     .iOS
-                    ////.AppBundle(AppPath) // Used to run a .app file on an ios simulator
 
-                     .InstalledApp(IpaBundleId) // Used to run a .ipa file on a physical ios device
+                    // .AppBundle(AppPath) // Used to run a .app file on an ios simulator
+                    .InstalledApp(IpaFileName) // Used to run a .ipa file on a physical ios device
```

Comment:
Android is named ApkPath, could this one be IpaPath accordingly? Or the other way around

Reply:
Sorry about this, I was wrong